Assembler.Core/Entities/BaseMessageInAssembly.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Assembler.Core.Enums;
+
+namespace Assembler.Core.Entities
+{
+    public abstract class BaseMessageInAssembly : BaseMessage
+    {
+        public ReleaseReason ReleaseReason;
+        public bool MiddleReceived;
+
+        protected BaseMessageInAssembly()
+        {
+            ReleaseReason = ReleaseReason.Unreleased;
+            MiddleReceived = false;
```

Comment:
Just a heads-up, you can also put a default value directly when declaring the property, i.e. 
```
public ReleaseReason ReleaseReason { get; set; } = ReleaseReason.Unreleased;
public bool MiddleReceived { get; set; } = false;
```
Up to you

Reply:
Cool!
I used it.